packages/create-eas-build-function/templates/typescript/package.json

```diff
@@ -12,6 +12,6 @@
   "devDependencies": {
     "@types/node": "20.11.0",
     "typescript": "^5.3.3",
-    "@expo/steps": "^1.0.55"
```

Comment:
Oh gosh, here too. Should we add a script to release-it to update this automatically on a new release?

Reply:
yeah I think so, but I believe we should do it in separate PR